android/src/main/kotlin/com/adyen/checkout/flutter/utils/ConfigurationMapper.kt

```diff
@@ -409,4 +407,17 @@ object ConfigurationMapper {
         cvc?.let { unencryptedCardBuilder.setCvc(it) }
         return unencryptedCardBuilder.build()
     }
+
+    fun InstantPaymentConfigurationDTO.mapToInstantConfiguration(context: Context): InstantPaymentConfiguration {
+        val environment = environment.fromDTO()
+        val instantPaymentConfigurationBuilder =
+            if (shopperLocale != null) {
+                val locale = Locale.forLanguageTag(shopperLocale)
+                InstantPaymentConfiguration.Builder(locale, environment, clientKey)
+            } else {
+                InstantPaymentConfiguration.Builder(context, environment, clientKey)
```

Comment:
This constructor is deprecated now, you can remove the context parameter. But if you're planning on migrating to the `CheckoutConfiguration` then you can ignore it for now.